packages/shared/src/types.ts

```diff
@@ -1,30 +1,6 @@
-import type { ApiError, User } from '@supabase/supabase-js';
-import { ErrorPayload } from './utils/errors';
+import type { CookieSerializeOptions } from 'cookie';
 
-export type UserFetcher = (url: string) => Promise<{
-  user: User | null;
-  accessToken: string | null;
-  error?: string | null;
-}>;
-
-export type UserState = {
-  user: User | null;
-  accessToken: string | null;
-  error?: ErrorPayload;
-  isLoading: boolean;
-  checkSession: () => Promise<void>;
-};
-
-export interface CookieOptions {
-  // (Optional) The Cookie name prefix. Defaults to `sb` meaning the cookies will be `sb-access-token` and `sb-refresh-token`.
-  name?: string;
-  // (Optional) The cookie lifetime (expiration) in seconds. Set to 8 hours by default.
-  lifetime?: number;
-  // (Optional) The cookie domain this should run on. Leave it blank to restrict it to your domain.
-  domain?: string;
-  path?: string;
-  // (Optional) SameSite configuration for the session cookie. Defaults to 'lax', but can be changed to 'strict' or 'none'. Set it to false if you want to disable the SameSite setting.
-  sameSite?: string;
-}
```

Comment:
I just tried this version, and this change broke builds or me, because the sveltekit helpers seemed to import it.

Reply:
SvelteKit helpers haven't been upgraded to work with v2 as yet.